### PR TITLE
Fix init script parameter bug

### PR DESCRIPTION
**Pull request by pmbuko**
#### Pull Request (PR) description
Change tag format to correctly add service name to init script Provides line and prevent stripping line break

#### This Pull Request (PR) fixes the following issues
#306

### CODE REVIEW

**Comment by dhoppe** [comment]
The tests fail because ```Provides``` is only set for Debian and Ubuntu:
* https://github.com/voxpupuli/puppet-kafka/blob/master/templates/init.erb#L5

File: spec/acceptance/producer_spec.rb [hunk 1]
```diff
@@ -130,14 +130,31 @@ class { 'kafka::producer':
         apply_manifest(pp, catch_changes: true)
       end
 
-      describe file('/etc/init.d/kafka-producer') do
+      describe file('/etc/init.d/kafka-producer'), if: (fact('operatingsystemmajrelease') =~ %r{(5|6)} && fact('osfamily') == 'RedHat') do
```
Comment (pmbuko) [inline spec/acceptance/producer_spec.rb:133]: kafka-producer service was missing the same os-specific spec tests that existed for the other services.

**Comment by bastelfreak** [comment]
thanks for the fix!

**Comment by dhoppe** [comment]
I think this fix was not necessary, because it only affects Debian and Ubuntu. 
* https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/service/upstart.rb#L19

All Debian and Ubuntu releases, supported by this module, already use Systemd.

**Comment by pmbuko** [comment]
> I think this fix was not necessary, because it only affects Debian and Ubuntu.
> 
> * https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/service/upstart.rb#L19
> 
> All Debian and Ubuntu releases, supported by this module, already use Systemd.

Ah, noted. I'm using an older version of the module and still have Ubuntu 14 hosts in production. :)

**Pull request merged**